### ISSUE TITLE
Some question about finetune ucf101 and hmdb51.

### ISSUE THREAD

**Issue by HXDreamChaser** [IBM/action-recognition-pytorch#6]
Thank you for sharing. Regarding the current work, I have some questions I hope you can answer. 
①How to deal with the data sets of UCF101 and HMDB51? What is the sampling frame rate when the video is processed into pictures, can you provide the relevant processing code and share more details?
②How did you fine-tune UCF101 and HMDB51 for model "I3D ResNet-50"? After I fine-tune the HMDB51 data set, the result is only 55%, and the result in the paper is 72%. The result of fine-tuning UCF101 is only 90%, which is 97.2% in the paper. Hope you can provide detailed parameter settings for fine-tune, if you do not mind, the code is better.
The detailed settings of my experiment are as follows:
I used 4 GPUs and trained for 45 epochs totally. The number of frames is 32. We use cosine annealing for learning rate decay, and the batchsize is 24(in the paper it is 48), mini-batch 6, and the initial learning rate is 0.005 (in the paper, it is 0.01). Do you have any suggestions for fine-tuning?

**Comment by marawangamal** [comment]
Hello, I am also interested in knowing the settings for the model transferability study. I was only able to achieve 63% on HMDB after using the I3D-ResNe50 model pretrained on kinetics.
